### DISCUSSION TITLE
getting below error while running the application in terminal

### DISCUSSION THREAD

**Discussion by dharmik3504** [arnav-kr/miniTV#3] [General]
![image](https://github.com/arnav-kr/miniTV/assets/77182459/7c2c6ace-d69d-452c-907b-345057352b69)

**Comment by arnav-kr** [comment]
Updating your nodejs to a higher version should do the fix, as it uses [undici fetch](https://undici.nodejs.org/#/?id=undicifetchinput-init-promise) under the hood which requires node js version `16.8+`.

So updating you version should work 

https://nodejs.dev/en/download/

latest is v20 and lts is v18
